target/linux/mpc85xx/image/p2041.mk

```diff
@@ -0,0 +1,16 @@
+define Device/freescale_p2041rdb
+  DEVICE_VENDOR := Freescale
+  DEVICE_MODEL := P2041RDB
+  DEVICE_DTS_DIR := $(DTS_DIR)/fsl
+  DEVICE_PACKAGES := kmod-hwmon-lm90 kmod-rtc-ds1307 \
+	kmod-gpio-pca953x kmod-eeprom-at24
+  BLOCKSIZE := 128k
+  KERNEL := kernel-bin | gzip | uImage gzip
+  SUPPORTED_DEVICES := fsl,P2041RDB
```

Comment:
If this is converted to lower-case as commented in the kernel patch, I'd change the device definition to fsl_p2041rdb and then drop the SUPPORTED_DEVICES here ...